library_generation/test/resources/integration/pr-description-golden.txt

```diff
@@ -0,0 +1,10 @@
+7a9a855287b5042410c93e5a510f40efd4ce6cb1
+feat: expose model_type in v1 processor, so that user can see the model_type after get or list processor version
+
+PiperOrigin-RevId: 603727585
```

Comment:
I think we can remove this in the PR description.

Reply:
This is part of the commit message and it is included in Owlbot PR.

Reply:
It is, but I don't think they provide much value and we don't want the release notes to include this message, can we try to parse the commit message and remove this part? Again, take a look at the existing [release notes](https://github.com/googleapis/google-cloud-java/releases/tag/v1.30.0), we should try to make our PR description as close as possible.

In addition, if we really want to keep the existing commit history, we can put it outside of the `BEGIN_COMMIT_OVERRIDE` and `END_COMMIT_OVERRIDE`.

Reply:
How about I only keep the first line of each commit message? The first line is usually the commit summary.